packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -78,7 +78,9 @@ def marathon_test_app(
                 'gracePeriodSeconds': 5,
                 'intervalSeconds': 10,
                 'timeoutSeconds': 10,
-                'maxConsecutiveFailures': 3
+                'maxConsecutiveFailures': 120  # ~20 minutes until restarting
+                # killing the container will rarely, if ever, help this application
+                # reach a healthy state, so do not trigger a restart if unhealthy
```

Comment:
@jgehrcke Oops, ben and I discussed this elsewhere and I forgot to update. Ben thought this was in relation to allowing tasks to be timed out in mesos (~10 minutes). Rather, this is to address a separate issue: restarting a container after TASK_RUNNING (as the health check will do) will not help the network interface come up (which is what is being waited for)